### ISSUE TITLE
Does multiple poll work ?

### ISSUE THREAD

**Issue by gabrielstuff** [mtrpcic/polljs#2]
I just tried :

``` javascript
Poll.start({
        name: "foo",
        interval: 1000,
        action: function() {
            console.log("foo")
        }
    });
Poll.start({
        name: "bar",
        interval: 5000,
        action: function() {
            console.log("bar")
        }
    });
```

only the last one is registered.

**Comment by mtrpcic** [comment]
Sorry for the issue, I should have more effectively tested PollJS.  I've fixed the bug in [this commit](https://github.com/mtrpcic/polljs/commit/01f211501f1dc12bd57072f5a2881e2e04659843), and bumped the version number to `0.3.0`.  The issue should be fixed now.  Please test in your application and close the issue.  If this issue isn't closed within 48 hours and I haven't heard back, I'll assume that my fix worked and I'll close it myself.

Thanks for using PollJS, I hope it makes your life easier :)

**Comment by mtrpcic** [comment]
One thing to note is that I just noticed that I neglected to update the minified version of the file.  It will be updated by the end of the night tonight.  Sorry for any inconvenience.

**Comment by gabrielstuff** [comment]
i'll keep you informed until end of the week :) sorry for that delay !

**Comment by mtrpcic** [comment]
As mentioned in a previous comment, this has been fixed and tested in PollJS v0.3.  If there are any further issues, please start a new Issue.  Thanks again for using PollJS and helping out by publishing issues.
